### PR TITLE
Fixes smashing people on tables

### LINKED ISSUE #28072: Glass tables don't break anymore.
#28034

Kevinz "Creates 3 bugs for every 1 fixed" 00 strikes again.

### PR DESCRIPTION
**Pull request by silicons**
woops
fixes #28072

### CODE REVIEW

**Comment by obscolene** [comment]
did you test this

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -195,7 +195,7 @@
 		check_break(M)
 
 /obj/structure/table/glass/proc/check_break(mob/living/M)
-	if(M.has_gravity() && M.mob_size > MOB_SIZE_SMALL && !M.movement_type & FLYING)
+	if(M.has_gravity() && (M.mob_size > MOB_SIZE_SMALL) && !(M.movement_type & FLYING))
```
Comment (ChangelingRain) [inline code/game/objects/structures/tables_racks.dm]: you shouldn't need to () `M.mob_size > MOB_SIZE_SMALL`

**Comment by kevinz000** [comment]
@obscolene 🤔

**Comment by Tacolizard** [comment]
@obscolene shut up filthy noncoder scum get off github normie

**Pull request merged**